app/styleguides/_default/components/subheader.css

```diff
@@ -35,11 +35,11 @@ $helvetica-stack: Helvetica, sans-serif;
   }
 
   .link {
+    opacity: 0;
```

Comment:
Should these changes be in a separate PR since they're a separate component?

Reply:
i can do a separate PR, but because is just order the property alphabetically and is just one mino change. i changed it in this branch